lib/sycamore/pyproject.toml

```diff
@@ -71,6 +71,7 @@ weaviate-client = { version = ">=4.6.1,<4.7", optional = true } # typechecking f
 
 # Local inference dependencies
 easyocr = { version = "^1.7.1", optional = true }
+paddleocr = { version = "^2.8.1", optional = true }
```

Comment:
When you install the paddleocr package, does it automatically install the cpu paddlepaddle, or do you have install it separately. (I think it's okay either way, just curious).

Reply:
No you'll have to install it. I wanted to leave it up to the user since they might want to install `paddlepaddle-gpu` instead depending on their machine.